src/relay.js

```diff
@@ -233,6 +233,23 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
           model.sequelize.literal('COUNT(1) OVER()'),
           'full_count'
         ]);
+      } else if (options.limit) {
+        // TODO: scope are not applied to the where clause!
+
+        const countQuery = model
```

Comment:
@mickhansen Do you have idea/tip about the query ?
I looked at [Model definition](https://github.com/sequelize/sequelize/blob/master/lib/model.js#L1280) in Sequelize and there is some methods that transform option before query:
- `this._conformOptions()`
- `this.$injectScope()`
- `this._expandIncludeAll()`
- `Utils.mapFinderOptions()`
- `this._paranoidClause()`

Is there a way to applied it to the option from outside from Sequelize ?
Furthermore is it necessary to clone `options.include` and `options.where` before passing it to `selectQuery` ?